staff/miquel-guevara/bash/README.md

```diff
@@ -0,0 +1,247 @@
+# BASH
+
+Command-line interface commands typical in BASH
+
+## git commit
+
+```sh
+$ git commit
+
+captura una instantánea de los cambios preparados en ese momento del proyecto
```

Comment:
Each command definition and examples should be written outside the back ticks 🧐

Reply:
Tot ok ja, gracies